brat/Brat/Compile/Hugr.hs

```diff
@@ -437,7 +438,7 @@ compileWithInputs parent name = gets compiled >>= (\case
           _ -> compileWithInputs parent outNode >>= \case
             Just calleeId -> do
               callerId <- addNode ("indirect_call(" ++ show calleeId ++ ")")
-                                  (OpCallIndirect (CallIndirectOp parent (FunctionType ins outs)))
+                                  (OpCallIndirect (CallIndirectOp parent (FunctionType ins outs bratExts {-[]-})))
```

Comment:
Should this be empty or not, c.f. `LoadFunction`?

Reply:
The extensions here should match the extensions of the callee, but we're overapproximating